docs/2022/reuse/updates/2022-06-02.md

```diff
@@ -53,3 +53,53 @@ GSoC 2022 weekly update
 
 - Project Meeting time for resue project - every tuesday 5:30 pm to 6:00 pm IST.
 - Need to update the weekly progress in [Tool](https://fossology.github.io/gsoc/docs/2022/)
+
+## Meeting 2
+*(June 7th, 2022)*
+
+GSoC 2022 Discussions - REUSE
+
+<!--truncate-->
+
+### Attendees
+
+- [Nicolas Toussaint](https://github.com/NicolasToussaint)
+- [Gaurav Mishra](https://github.com/GMishx)
+- [Shaheem Azmal M MD](https://github.com/shaheemazmalmmd)
+- [Vivek Kumar](https://github.com/viv9k)
+- [Kaushlendra Pratap](https://github.com/Kaushl2208)
+- [Rohit Pandey](https://github.com/rohitpandey49)
+
+### Discussions
+
+- **How should licenses and copyrights be applied to files without licenses or copyrights?**
+
+    - Use GPL 2.0-only license for such files without copyright.
+
+- **Do I need to create a Wiki Page for the REUSE Standard?**
+
+    - Yes. You can implement it in 2 ways: i) By creating a separate wiki page ii) By adding in `fossology/CONTRIBUTING.md`
+
+- **Do I need to add the current year to the copyright?**
+
+    - No. Since we are not changing any logic in the file, we shouldn't change the copyright year.
+
+- **How to add license and copyright in JSON files?**
+
+    - JSON is for data only, and if you include a comment, then it will be data too. The `file-name.json.license` file needs to be created to add the license and copyright.
+
+- **Can we have more than one license for a file?**
+
+    - Yes, it’s possible. You can use SPDX syntax to define them.
+    For example: 
+    SPDX-License-Identifier: (GPL-2.0 OR MIT)
+    SPDX-License-Identifier: (LGPL-2.1 AND BSD-2-CLAUSE)
+
+- **Should I change the year ranges?**
+
+    - Concerning the year ranges, we should not change the existing year ranges(and not change the end year if no substantial modification was done in the file).
+
+- Do not modify testdata files.
+### Conclusion and Further Plans
+
+- Implement discussed solution in upcoming week.
```

Comment:
Why am I seeing updates from Rohit Pandey in your branch? Any idea @dushimsam 

Please remove if irrelevant.

Reply:
It was caused by the new changes which were not present locally.
So I needed to pull them from this main repository first.

Reply:
That's why we recommend rebasing instead of pulling :-)

Try using `git pull --rebase origin main`

Reply:
alright got it, sir.